source/guides/views/layouts.md

```diff
@@ -66,6 +66,37 @@ end
 
 If we want to turn off this feature entirely, we can set `layout nil` into the application's configuration.
 
+## Using Multiple Templates
+
+Sometimes it's useful to have more than one template. For example, if the `application.erb` template contains an HTML structure you don't want for, say, a login page - a separate `login.html.erb` template can be used.
+
+Assuming you have created a `home#login` action, you create the `login.html.erb` template right next to your default `application.html.erb` in `/web/templates/`.
```

Comment:
I'm not sure we want to use `home#login` as an example, since it's not a RESTful. Instead, we might want to encourage something like `user_sessions#new` using this `login` layout.